testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/persistence/YamlExtensionTestCase.java

```diff
@@ -42,6 +42,7 @@
  *
  * @author Emmanuel Hugonnet (c) 2021 Red Hat, Inc.
  */
+@org.junit.Ignore("WFCORE-6680")
```

Comment:
@pferraro please rebase as this test is fixed

Reply:
Will do.